### PR TITLE
store/tikv: add more probe functions for tests

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
Add more probe utilities in `test_probe.go` so later we can move tests files into `tests` directory.

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- add new function and utilities. (most of them are not used yet)

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/test_probe.go [hunk 1]
```diff
@@ -34,6 +36,43 @@ func (s StoreProbe) NewLockResolver() LockResolverProbe {
 	return LockResolverProbe{LockResolver: newLockResolver(s.KVStore)}
 }
 
+// GetTimestampWithRetry returns latest timestamp.
+func (s StoreProbe) GetTimestampWithRetry(bo *Backoffer, scope string) (uint64, error) {
```
Comment (AndreMouche) [inline store/tikv/test_probe.go:40]: Where do we need this function? Or it is just prepareing for the next PRs?

Comment (disksing) [reply store/tikv/test_probe.go:40]: Yes. For later PR.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by sticnarf — APPROVED** [review]
LGTM

**Pull request merged**